### DISCUSSION TITLE
Verbose Installation option for Console

### DISCUSSION THREAD

**Discussion by heath-toby** [wstyres/Zebra#733] [Ideas]
Hello. It would be great if in Zebra, there were to be a setting where we can get all the info from the command line when installing packages. At the moment, we just get downloading packages and then performing actions. I think it would be nice to have everything if we want, starting from "Reading package lists... Building dependency tree..." This could be good if one wanted to see if there were any errors during installation. Thanks.

**Comment by wstyres** [comment]
All of the APT output is already shown during the "Performing Actions" section, no dependency trees are needed as they're calculated before the console even appears. Errors will show up in the console if necessary but if there is something else wrong with the output this is more of a bug than a suggestion

**Comment by heath-toby** [comment]
Oh, because for me all I see is performing actions and to its right, nothing else. In fact sometimes, upon installation completion or failure, I just see downloading and then nothing else.
Sorry if that is a bug report, not a suggestion...

**Comment by wstyres** [comment]
Could you show a screenshot of the package installation screen?

**Comment by heath-toby** [comment]
Here's a link, this email client which works with my email address cannot upload photoes...

https://share.icloud.com/photos/0RS_5X2YxRaSRFRdthqbfumpw

**Comment by wstyres** [comment]
That looks normal to me, what are you requesting be added?

**Comment by heath-toby** [comment]
So for me, I just saw downloading, done, and then that was it. I was saying it would be nice for a setting or something to show more info, such as selecting unselected package or anything else. Some have tags, say, after the unpacking part, saying something like "Please respring your device". This is why it would be nice to see everything that happens after the package downloads.On 22/12/2019 at 18:45, Wilson Styres wrote:

**Comment by wstyres** [comment]
So just a more verbose installation process?

**Comment by heath-toby** [comment]
Yes. Thanks, and sorry for being so confusing.

**Comment by wstyres** [comment]
Alright, we can look into this for a future release.

**Comment by heath-toby** [comment]
Thank you very much.
